webpify.js

```diff
@@ -0,0 +1,183 @@
+const sharp = require('sharp');
+const glob = require('glob');
+const path = require('path');
+const fs = require('fs').promises;
+
+// Array to store original file paths
+const originalFiles = [];
+const convertedFiles = [];
+
+async function deleteOriginalFiles() {
+    console.log('\nDeleting original files...');
+    for (const conversion of convertedFiles) {
+        if (conversion.status === 'converted' || conversion.status === 'skipped') {
+            try {
+                await fs.unlink(conversion.original);
+                console.log(`Deleted: ${conversion.original}`);
+            } catch (error) {
+                console.error(`Error deleting ${conversion.original}:`, error);
+            }
+        }
+    }
+    console.log('Original file deletion complete!');
+}
```

Comment:
This is a very nice suggestion, but, I will have just one small nitpick. Can you add the same for replacing the file references? So if the file could not be converted, the references to it would not change.

Reply:
I am just realizing that this is not really necessary.
After the conversion, the build process runs, and if something during the conversion fails, the build will fail because of a missing module error while importing most images.
I will add a small bit of code, which does not deleted skipped / unconverted images, though.